src/portal/azext_portal/generated/__init__.py

```diff
@@ -0,0 +1,12 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Comment:
I suggest not adding `__path__ = __import__('pkgutil').extend_path(__path__, __name__)` to chose one of Python Namesapce, leave it with License is ok.

Reply:
I just use the same one as in logic. Will remove this line in next commit